File: tcmb-currency-app-ui/src/app/currency-list/Currency.ts
export class Currency{
    currencyCode: string;
    currencyName: string;
    forexBuying: number;
    forexSelling: number;
    banknoteBuying: number;
    banknoteSelling: number;


    constructor(
        _currencyCode: string,
        _currencyName: string,
        _forexBuying: number,
        _forexSelling: number,
        _banknoteBuying: number,
        _banknoteSelling: number
    ) {
        this.currencyCode = _currencyCode,
        this.currencyName = _currencyName,
        this.forexBuying=_forexBuying,
        this.forexSelling=_forexSelling
        this.banknoteBuying=_banknoteBuying
        this.banknoteSelling=_banknoteSelling
    }

}

File: tcmb-currency-app/src/main/java/com/tcmb/entity/Currency.java
package com.tcmb.entity;

import lombok.Data;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

import javax.persistence.*;


@Data
@Setter
@Getter
@ToString
@Entity
public class Currency {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String date;

    private String currencyCode;

    private String currencyName;

    private Double forexBuying;

    private Double forexSelling;

    private Double banknoteBuying;

    private Double banknoteSelling;
}
